sleap_roots/tips.py

```diff
@@ -15,3 +17,47 @@ def get_tips(pts):
     # Get the last point of each instance. Shape is (instances, 2)
     tip_pts = pts[:, -1]
     return tip_pts
+
+
+def get_primary_depth(pts: np.ndarray):
+    """Get primary root tip depth.
+
+    Args:
+        pts: primary root landmarks as array of shape (1, point, 2)
+
+    Returns:
+        Primary root tip depth (location in y-axis).
+    """
+    # get the last point of primary root, if invisible, return nan
+    if pts[:, -1].any() == np.nan:
+        return np.nan
```

Comment:
Test for this case?

Reply:
Already had a test "test_get_primary_depth_nan(pt_nan_tip)". Did I misunderstand your comment?